### DISCUSSION TITLE
Is there a limit of number of validators for polygon edge?

### DISCUSSION THREAD

**Discussion by taylorkchan** [0xPolygon/polygon-edge#558] [Q&A] [answered]
Hi I am wondering is there a limit of number of validators and bootnode for polygon edge?

**Comment by zivkovicmilos** [comment, marked as answer]
Hey @taylorkchan,

Depends on how you define _limit_.

If we're talking about technical limitations, Polygon Edge doesn't explicitly have the cap on the number of nodes you can have in a network. You can set connection caps (inbound / outbound connection counts) on a per-node basis.

If we're talking about practical limitations, you're gonna see a more degraded performance with a 100 node cluster than with a 10 node cluster. By increasing the number of nodes in your cluster, you increase the communication complexity and just the networking overhead in general. It all depends on what kind of network you're running, and what kind of network topology you have.
